src/app/pages/admin/twins/twins.component.html

```diff
@@ -9,14 +9,14 @@
                (input)="searchThing($event.target.value)">
       </div>
       <div class="col-md-2">
-        <button nbButton size="small" status="success"
+        <button nbButton size="small" class="buttonBlue"
```

Comment:
I'm no expert by any means, but hyphens are usually used instead of camelcase for class names. The main reason for this is that HTML is case insensitive, so different class names can end up being treated equally.

Reply:
right, let me fix this